examples/article.amp.html

```diff
@@ -271,8 +271,8 @@
             src="img/hero@1x.jpg"
             srcset="img/hero@1x.jpg 1x, img/hero@2x.jpg 2x"
             layout="responsive" width="360" placeholder
-            alt="Curabitur convallis, urna quis pulvinar feugiat, purus diam posuere turpis, sit amet tincidunt purus justo et mi."
-            height="216" on="tap:headline-img-lightbox" role="img" tabindex="0">
+            alt="Two chairs on a lake shore with big pine trees"
+            height="216" on="tap:headline-img-lightbox" tabindex="0" role="img">
```

Comment:
Let's remove `role="img"` and add `opens in a lightbox` to the end of the `alt`.